app/src/main.rs

```diff
@@ -104,8 +96,39 @@ extern "C" fn power_callback() {
 
 /// Instructs the system to shut down cleanly.
 pub fn shutdown() -> ! {
-    unsafe {
-        STM_ShutdownToStandby();
+    // unsafe {
+    //     STM_ShutdownToStandby();
+    // }
+    unsafe { exit(0) };
+    // core::unreachable!()
+}
+
+struct InputManager {
```

Comment:
Again, sorry if I'm nitpicky and I feel like you've explained this to me before, but... :see_no_evil: 
Wouldn't it be neater to separate these things into separate modules like `input.rs`?

Reply:
Good idea!